### PR TITLE
Add Lambda Runtime Support for Python_3.11

### PR DESCRIPTION
**Pull request by vchikoti1998**
## Types of changes

- [ ] Bug fix (non-breaking change which fixes an issue)
- [ x] New feature (non-breaking change which adds functionality)

## Description
 Lambda Runtime Support for Python_3.11

## Checklist
- [ ] My code follows the code style of this project
- [ ] I have added tests to cover my changes
- [x ] A short description of the change has been added to the **[CHANGELOG](https://github.com/aws/aws-toolkit-jetbrains/blob/master/CONTRIBUTING.md#contributing-via-pull-requests)** if the change is customer-facing in the IDE.
- [ ] I have added metrics for my changes (if required)
 
## License
I confirm that my contribution is made under the terms of the Apache 2.0 license.

### CODE REVIEW

File: gradle/libs.versions.toml [hunk 1]
```diff
@@ -2,7 +2,7 @@
 apacheCommons = "2.8.0"
 assertJ = "3.20.2" # Upgrading leads to SAM errors: https://youtrack.jetbrains.com/issue/KT-17765
 # match with <root>/settings.gradle.kts
-awsSdk = "2.20.55"
+awsSdk = "2.20.111"
```
[the remark this answers is not in the document]
Comment (vchikoti1998) [reply gradle/libs.versions.toml:5]: sure, will match with same version.

File: testdata/samProjects/image/python3.11/hello_world/requirements.txt
Comment (manodnyab) [inline testdata/samProjects/image/python3.11/hello_world/requirements.txt:1]: Is this file required?

Comment (vchikoti1998) [reply testdata/samProjects/image/python3.11/hello_world/requirements.txt:1]: I believe so

**Review by manodnyab — APPROVED** [review]
